client/src/partials/reports/modals/accounts_chart.config.js

```diff
@@ -12,6 +12,7 @@ function AccountChartController($state, ModalInstance, Notify, Languages, SavedR
   vm.cancel = ModalInstance.dismiss;
   vm.report = report;
 
+  vm.reportTitleKey = 'REPORT.CHART_OF_ACCOUNTS';
```

Comment:
Ideally this should be taken from the instance passed into the modal from the controller vs. hard coded here. I believe https://github.com/IMA-WorldHealth/bhima-2.X/pull/838 implements this as an example.